themes/default/content/docs/clouds/kubernetes/get-started/deploy-changes.md

```diff
@@ -14,94 +14,113 @@ aliases:
 - /docs/get-started/kubernetes/deploy-changes/
 ---
 
-Now let's deploy our changes.
+Deploy the stack changes.
 
 ```bash
 $ pulumi up
 ```
 
 Pulumi computes the minimally disruptive change to achieve the desired state described by the program.
 
-```
-Previewing update (dev):
+```bash
+Previewing update (dev)
+
+View in Browser (Ctrl+O): https://app.pulumi.com/diana-pulumi-corp/quickstart/dev/previews/1aed8afe-5f92-48bc-bbb4-0249471150c6
+
+     Type                              Name            Plan
+     pulumi:pulumi:Stack               quickstart-dev
+ +   ├─ kubernetes:apps/v1:Deployment  nginx           create
+ +   ├─ kubernetes:core/v1:Service     nginx           create
+ -   └─ kubernetes:apps/v1:Deployment  app-dep         delete
 
-     Type                        Name            Plan
-     pulumi:pulumi:Stack         quickstart-dev
- +   └─ kubernetes:core:Service  nginx           create
 
- Outputs:
-  + ip  : output<string>
-  - name: "nginx-xw231xdt"
+Outputs:
+  + ip  : "10.96.0.0"
+  - name: "app-dep-b7413dae"
 
 Resources:
-    + 1 to create
-    2 unchanged
+    + 2 to create
+    - 1 to delete
+    3 changes. 1 unchanged
 
-Do you want to perform this update?
+Do you want to perform this update?  [Use arrows to move, type to filter]
   yes
 > no
   details
 ```
 
+Select `yes` using the arrows and hit enter to update the resources in Kubernetes.
+
 Pulumi will create the service since it is now defined in the program.
 
-```
+```bash
```

Comment:
I'd probably suggest either leaving these as-is or using `plain`, since the output isn't Bash. (This will suppress the copy buttons of course, so if the intent was to make these snippets more easily copyable, we could definitely add support for plain test snippets to the copy-button script.)

Reply:
They don't need to be copyable as it's just terminal output. I did `bash` for the color formatting.  I'll update it as suggested :)